src/pages/index.astro

```diff
@@ -1,7 +1,27 @@
 ---
 import Layout from "../layouts/Layout.astro";
+import Card from "../components/Card.astro";
 ---
 
 <Layout>
-  <h1 class="font-sans text-4xl">nunito</h1>
+  <div class="flex gap-8">
+    <Card
+      class="bg-[#9AD3CC]"
```

Comment:
Please use a tailwind variable color like `bg-yellow`. To achieve this goal you need to add a custom color in the `tailwind.config.cjs`.

Reply:
Here you can find some documentation [Adding additional colors](https://tailwindcss.com/docs/customizing-colors#adding-additional-colors).

Reply:
I replaced arbitrary values